### PR TITLE
Add exclusion relationships to WI incentives.

### PR DESCRIPTION
**Pull request by hblumberg**
This PR handles the three incentives that have non-restricted and income-qualified versions.

## Links
- _Asana_ https://app.asana.com/0/1206629812394927/1206842549215196/f

## Checklist
- [x] This PR includes automated tests where possible.
- [x] Relevant READMEs, scripts, etc. in this repo have been updated if needed.
- [x] Relevant runbooks and ADRs in Notion have been updated if needed.
- [x] Checked that no new errors are introduced in the console or terminal

## Test Plan
* `yarn test`

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -60,6 +60,20 @@ All valid records according to our schema will be written to the location you sp
 Note that even after running these, there are still files you must edit manually to get the JSON in. Follow a recent CL example.
 Eg: https://github.com/rewiringamerica/api.rewiringamerica.org/pull/209/files
 
+### Incentive Relationships
```
Comment (oyamauchi) [inline scripts/README.md:63]: This stuff is also documented in `docs/relationships-README.md`, which I get is not super discoverable. It's also a bit more abstract and not focused on the sheet-to-JSON process.

Could you either fold this info into that doc, or the other way around, or whatever you think is best?

**Review by oyamauchi — APPROVED** [review]
The actual change looks fine but we need to sort out the docs thing.

File: scripts/README.md (hunk 1 above)
Comment (hblumberg) [reply scripts/README.md:63]: Oops, I totally missed that -- thank you! What do you think of linking to that README here?

Comment (oyamauchi) [reply scripts/README.md:63]: I think that's fine -- `docs` can be for more in-depth stuff, and this file for practical sheet-to-JSON info.

**Pull request merged**